src/main/java/core/basesyntax/DataConverterImpl.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> inputData) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        for (String line : inputData) {
+            if (line.startsWith("type")) {
+                continue;
+            }
+            String[] parts = line.split(",");
+            String operationCode = parts[0];
+            String fruit = parts[1];
+            int quantity = Integer.parseInt(parts[2]);
```

Comment:
You should handle the case where `parts[2]` is not a valid integer. If it's not, `Integer.parseInt()` will throw a `NumberFormatException`. You should catch this exception and throw a more informative one in its place.

Reply:
changes have been made